<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogIn</class>
 <widget class="QMainWindow" name="LogIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Heading</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;INICIAR SESIÓN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sylfaen</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Usuario:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sylfaen</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Contraseña:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_user">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Usuario:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_password">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Contraseña:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_login">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Small</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 255, 187);
selection-color: rgb(28, 255, 187);
border-color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>ENTRAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
